src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -213,7 +216,7 @@ internal static WorkerInitResponse WorkerInitRequestHandler(WorkerInitRequest re
             response.Capabilities.Add("WorkerStatus", bool.TrueString);
             response.Capabilities.Add("HandlesWorkerTerminateMessage", bool.TrueString);
             response.Capabilities.Add("HandlesInvocationCancelMessage", bool.TrueString);
-
+            response.Capabilities.Add("EnableUserCodeException", enableUserCodeException.ToString());
```

Comment:
If the worker option property value is false, should we even send this capability entry? Looks like host code does not check the value of the entry, but checks whether this capability entry is present or not.

https://github.com/Azure/azure-functions-host/blob/ec3c2d1049351870bbf611760520cf1e48075c8f/src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs#L786

Reply:
That pattern is used for all Boolean type capabilities. 
https://github.com/Azure/azure-functions-host/blob/ec3c2d1049351870bbf611760520cf1e48075c8f/src/WebJobs.Script.Grpc/MessageExtensions/GrpcMessageConversionExtensions.cs#L328-L356

Reply:
Ooh that's a good point! Yes in that case we shouldn't send the capability if the option is false.